samples/javascript_nodejs/53.teams-messaging-extensions-action-preview/adaptiveCardHelper.js

```diff
@@ -69,6 +71,15 @@ class AdaptiveCardHelper {
                     placeholder: 'Option 3 here',
                     type: 'Input.Text',
                     value: option3
+                },
+                { type: 'TextBlock', text: 'Do you want to send this card on behalf of User?' },
+                {
+                    choices: [{ title: 'True', value: 'true' }, { title: 'False', value: 'false' }],
```

Comment:
```suggestion
                { type: 'TextBlock', text: 'Do you want to send this card on behalf of the User?' },
                {
                    choices: [{ title: 'Yes', value: 'true' }, { title: 'No', value: 'false' }],
```

I haven't tried this in the sample, but I'm assuming that since `value` stays `true/false`, this would still work.